test/transaction.ts

```diff
@@ -61,6 +63,9 @@ describe('Transaction', () => {
     request_() {},
     projectId: PROJECT_ID,
     namespace: NAMESPACE,
+    addDatabaseIdToRequest(datastore: Datastore, reqOpts: RequestOptions) {
```

Comment:
it's still unclear to me why we can't instead add `databaseId: DATABASE_ID` here similar to these other values (projectId, namespace). Why is database id treated differently?

Reply:
I think I misunderstood the question from before. We could add **databaseId: SECOND_DATABASE_ID** here. DATABASE_ID is used lower on the stack than `request` and request is mocked out so it won't do anything, but it won't hurt either.

Reply:
I cherry picked the changes with test parameterization now so that we run tests against both the default database and the second database.